www/rounds.php

```diff
@@ -3,11 +3,24 @@
 require_once '../lib/Util.php';
 
 $rounds = Model::factory('Round')
-        ->order_by_desc('start')
+        ->order_by_desc('duration')
         ->find_many();
 
 $user = Session::getUser();
 
+
+const STATUS_CLASSES = [
+  Round::STATUS_EXPIRED => 'bg-danger',
```

Comment:
This seems a bit drastic since, in time, most of the table will consist of expired rounds.